doc/whats_new/v0.22.rst

```diff
@@ -460,6 +460,13 @@ Changelog
   with a target matrix `Y` in which the first column was constant.
   :issue:`13609` by :user:`Camila Williamson <camilaagw>`.
 
+:mod:`sklearn.metrics`
+..................................
+
+- |API| Introduced new scorer neg_brier_score_loss to better reflect real
```

Comment:
I'd prefer something like:

```
- |API| scoring='neg_brier_score_loss' should be used instead of
  scoring='brier_score_loss' which is now deprecated.
```

I think the effect on users is clearer.